scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/wson.yaml

```diff
@@ -396,3 +396,62 @@ WSON-12-1EP_4x4mm_P0.5mm_EP2.6x3mm:
   #suffix: '_PullBack'
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
+
+Texas_R-PWSON-N8:
+  device_type: 'WSON'
+  library: Package_SON
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://www.ti.com/lit/ds/symlink/tpd3f303.pdf#page=15'
+  ipc_class: 'qfn' #'qfn' | 'qfn_pull_back'
+  custom_name_format: 'Texas_R-PWSON-N{pincount}_EP{ep_size_x:g}x{ep_size_y:g}mm{vias:s}'
```

Comment:
I am not sure if the manufacturer prefix is needed here. The rule for that is here:
https://kicad-pcb.org/libraries/klc/F2.3/

The current library has both prefixed and non prefixed `WSON` parts. It seems very inconsistent.
I guess having the custom format here is okay. So lets keep it.